Formula/rhit.rb

```diff
@@ -0,0 +1,50 @@
+class Rhit < Formula
+  desc "Nginx log explorer"
+  homepage "https://dystroy.org/rhit/"
+  url "https://github.com/Canop/rhit/archive/refs/tags/v1.5.3.tar.gz"
+  sha256 "fc2bd613a9ded70906f4c2af67a1540ffe6de165efe6aefeffaea97ceed76a82"
+  license "MIT"
+  head "https://github.com/Canop/rhit.git"
```

Comment:
```suggestion
  head "https://github.com/Canop/rhit.git", branch: "main"
```

Reply:
I'm gonna write an audit for this thing...